ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-ref.yaml

```diff
@@ -1,9 +1,6 @@
  ref:
   as: cucushift-upgrade-toimage
-  from_image:
-    namespace: ci
-    name: verification-tests
-    tag: latest
+  from: ansible
```

Comment:
Per https://github.com/openshift/release/pull/30893/files#r935271309, can we install ansible and jq in the `verification-tests` image

Reply:
Good idea! @liangxia can we do it^?

Reply:
We can get it merge to unblock rhel upgrade tests. Also log an jira issue to track the followup optimize.
@jianlinliu  @shellyyang1989   WDYT ?

Reply:
@liangxia It's fine with me! I just made an update and let's wait for the [rhel job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/30893/rehearse-30893-periodic-ci-openshift-openshift-tests-private-release-4.11-nightly-4.11-upgrade-from-stable-4.10-aws-ipi-proxy-workers-rhel8/1556465791358996480) result.

Reply:
@liangxia The rhel upgrade [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/30893/rehearse-30893-periodic-ci-openshift-openshift-tests-private-release-4.11-nightly-4.11-upgrade-from-stable-4.10-gcp-ipi-workers-rhel8/1556990978110263296) passed. It's ready for merging.